src/backend/transforms/interfaces.ts

```diff
@@ -9,7 +19,7 @@ export interface JiraPullRequest {
 	}
 }
 
-export interface JiraBuild {
+interface JiraBuild {
```

Comment:
Should probably still make this discoverable even if they're not being used currently.